src/main/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcher.java

```diff
@@ -56,6 +56,13 @@ public String getName() {
         try {
             if (importer.isRecognizedFormat(new BufferedReader(new StringReader(response)))) {
                 parserResult = importer.importDatabase(new BufferedReader(new StringReader(response)));
+            } else {
+                // For displaying An ErrorMessage
+                BibEntry errorBibEntry = new BibEntry();
+                errorBibEntry.setField("html_representation", "An error occured. We are sorry for that.");
```

Comment:
Can you use `Localiztaion.lang("Error_while_fetching_from_%0", "Mr.DLib")` instead of `An error occured....`.